src/style.css

```diff
@@ -1,3 +1,78 @@
+@media (min-width: 1024px) or (max-width: 1200px) {
+  header {
+    min-width: 1024px;
+    max-width: 1200px;
+  }
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  margin: auto;
+}
+
+.home_page {
+  height: 40px;
+}
+
+.nav {
+  display: flex;
+  flex-direction: row-reverse;
+  align-items: flex-start;
+  justify-content: space-between;
+
+}
+
+.nav_link {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-transform: uppercase;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  height: 60px;
+  margin: 0 0 0 20px;
+  padding: 0;
+}
+
+.nav_link1 {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+}
+
+a {
+  color: #00acdc;
+}
+
+a:visited {
+  color: #00acdc;
+}
+
+.nav_link:hover {
+  cursor: pointer;
+  color: #00acdc;
+}
+
+.is-active {
+  padding: 23px 0 0;
```

Comment:
The padding property in .is-active selector should not be applied only to the top. If you want to apply it only to the top, use 'padding-top', not 'padding'.